File: CSHARP/CSHARP/Model/Rezultat.cs
using System;
namespace CSHARP.Model
{
    public class Rezultat
    {
        public int prvi { get; set; }
        public int drugi { get; set; }
        public int treci { get; set; }


        public bool isPocetak()
        {
            return prvi == 0 && drugi == 0 && treci == 0;
        }

    }
}

File: Kotlin/src/main/kotlin/model/Rezultat.kt
package model

/**
 * @author Matija Skrletović
 **/

class Rezultat(var prvi: Int, var drugi: Int, var treci: Int = 0) {

    fun isPocetak(): Boolean {
        return prvi == 0 && drugi == 0 && treci == 0
    }


}